docs/search/criteria_reference/catalogidentifier_criterion.md

```diff
@@ -0,0 +1,33 @@
+# CatalogIdentifier Criterion
+
+The `CatalogIdentifier` Search Criterion searches for catalogs by the value of their identifier.
```

Comment:
As [there can be only one](https://doc.ibexa.co/en/latest/pim/catalog_api/#get-catalog), maybe singular can remind that an identifier is unique. 

```suggestion
The `CatalogIdentifier` Search Criterion searches for a catalog by the value of its identifier.
```